duet/dashboards/mobile_android_country.dashboard.lookml

```diff
@@ -249,31 +248,33 @@
     title: Event Installs
     note_state: collapsed
     note_display: above
+    note_text: The count of install events as reported in the Play Store. It includes
+      re-installs but not pre-installs on Google devices.
     merged_queries:
     - model: duet
       explore: mobile_android_country
       type: table
-      fields: [mobile_android_country.join_field, mobile_android_country.event_installs]
+      fields: [mobile_android_country.join_field, mobile_android_country.first_time_installs]
       fill_fields: [mobile_android_country.join_field]
       filters:
         mobile_android_country.period_offset: '0'
-      sorts: [mobile_android_country.event_installs desc]
+        country_buckets.bucket: ''
+      sorts: [mobile_android_country.first_time_installs desc]
       limit: 500
       join_fields: []
     - model: duet
       explore: mobile_android_country
       type: table
-      fields: [mobile_android_country.join_field, mobile_android_country.event_installs]
+      fields: [mobile_android_country.join_field, mobile_android_country.first_time_installs]
       fill_fields: [mobile_android_country.join_field]
       filters:
         mobile_android_country.period_offset: '1'
-      sorts: [mobile_android_country.event_installs desc]
+        country_buckets.bucket: ''
```

Comment:
@acmiyaguchi I noticed this filter is a new addition. In the UI when I was making changes it says this filter is required so I didn't remove it. When I validated the LookML everything checked out and when I filtered on countries it seems to work okay. Just wanted to call it out in case it is unnecessary or needs to be removed.

Reply:
It's probably fine, interesting that it got changed though.